emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -440,3 +440,15 @@ def update_data(user_id, key, obj_id, data):
         logging.debug("updating entry %s into timeseries" % new_entry)
         edb.save(ts.get_timeseries_db(key), new_entry)
 
+    def count_data(self, key, extra_query_list):
```

Comment:
nit: is `count_data` really the best term? we have `find_entries`, so we might want to make this more consistent instead of coming up with Yet Another Name.